content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -114,3 +114,143 @@ The scheduler (through the _VolumeZonePredicate_ predicate) also will ensure tha
 If `PersistentVolumeLabel` does not support automatic labeling of your PersistentVolumes, you should consider
 adding the labels manually (or adding support for `PersistentVolumeLabel`). With `PersistentVolumeLabel`, the scheduler prevents Pods from mounting volumes in a different zone. If your infrastructure doesn't have this constraint, you don't need to add the zone labels to the volumes at all.
 
+## node.kubernetes.io/windows-build {#nodekubernetesiowindows-build}
+
+Example: `node.kubernetes.io/windows-build=10.0.17763`
+
+Used on: Node
+
+When the kubelet is running on Microsoft Windows, it automatically labels its node to record the version of Windows Server in use.
+
+The label's value is in the format "MajorVersion.MinorVersion.BuildNumber".
+
+## service.kubernetes.io/headless {#servicekubernetesioheadless}
+
+Example: `service.kubernetes.io/headless=""`
+
+Used on: Service
+
+The control plane adds this label to an Endpoints object when the owning Service is headless.
+
+## kubernetes.io/service-name {#kubernetesioservice-name}
+
+Example: `kubernetes.io/service-name="nginx"`
```

Comment:
Because `nginx` is (can be) a load-balancing solution, I suggest:
```suggestion
Example: `kubernetes.io/service-name="mywebapp"`
```

Is that better?